src/GitHook/Command/Context/CommandContext.php

```diff
@@ -61,35 +61,108 @@ public function getCommandConfig($commandName)
      *
      * @return \GitHook\Command\Context\CommandContextInterface
      */
-    public function setFile($file)
+    public function setFile(string $file): CommandContextInterface
     {
         $this->file = $file;
 
         return $this;
     }
 
     /**
+     * {@inheritdoc}
+     *
      * @return string
      */
-    public function getFile()
+    public function getFile(): string
     {
+        if (strpos($this->file, './') === 0) {
+            $this->file = $this->getProjectPath() . substr($this->file, 2);
+        }
+
         return $this->file;
     }
 
+    /**
+     * @return bool
+     */
+    public function isModuleFile(): bool
```

Comment:
What about standalone ones?
i would directly look for the config via fallback.

Reply:
I recommend using the same logic the release app does:
https://github.com/spryker/release-app/blob/master/src/Integration/Task/PhpstanIntegrationTask.php#L71-L73

Those are the possible locations for finding the root files you are looking for.

Reply:
No need to as this tool uses the file path. No need to look up something which is there or not.

Reply:
Why do we need this method then?

Reply:
Its needed to check if the current file belongs to a module or not.

Reply:
How does that when you search only for 'Bundles'? Since other standalone ones would not be able to use this method here, and that makes this thing weird.
Why not just having a method like outlined above to get the root dir, check for the file and just fallback to default if not found. The info if module or not should not be relevant for this, should it?

Reply:
GitHook so far is not used in standalone module AFAIK. What do you think about to go back dir by dir until we can find a phpstan.json? If we do not find one we could then use the default.

That would work in all cases.

Anyway, this can be done after we merged this back as currently no file in any module is checked by this tool.

Reply:
I can fix the test and check if I can add the fallback thing to make it work in all cases.

Reply:
I wouldnt travel back in dirs automatically, I would use the approach of the release app, as that covers all relevant cases for us.
But sure, can also be done after the merge as separate improvement.